doc/logicalNameMapper.dox

```diff
@@ -278,8 +278,11 @@ those parts individually without having to deal with bit masks yourself.
 
 Parameters:
 - <code>shift</code>: Position of the first bit of the mask
-- <code>numberOfBits</code>: Length of the mask
-
+- <code>numberOfBits</code>: Length of the mask. Cannot be larger than 32.
+- <code>fractionalBits</code>: Optional, how many of the bits are representing the fractional part of
+  a fixed point value. Defaults to <code>0</code>. See ChimeraTK::FixedPointConverter() for details.
+- <code>signed</code>: Optional, whether the type represented by numberOfBits is signed. Defaults to
+  <code>false</code>
```

Comment:
1) Because it basically was "unsigned" for the original version of the code
2) The old test worked by accident using signed values since there was no interpretation attached at all and it just happened to do the right thing - so instead of changing the test I opted for just declaring the bits as signed